utils/utils.go

```diff
@@ -199,6 +215,22 @@ func pathToRegExp(localPath string) string {
 	return "^" + localPath + "$"
 }
 
+func antPatternToRegExp(localPath string) string {
+	var specialChars = []string{".", "^", "$", "+"}
+	for _, char := range specialChars {
+		localPath = strings.Replace(localPath, char, "\\"+char, -1)
+	}
```

Comment:
Since the above code line are duplicated in this function and the  ```wildcardPathToRegExp``` function, let's move then into a new private ```replaceSpecialChars``` function, and reuse it.